src/test/java/com/wikia/webdriver/common/templates/TemplateNoFirstLoad.java

```diff
@@ -6,6 +6,5 @@
 public class TemplateNoFirstLoad extends NewTestTemplate {
 
   @Override
-  protected void loadFirstPage() {
-  }
+  protected void loadFirstPage() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)